sklearn/mixture/tests/test_gmm.py

```diff
@@ -309,7 +309,7 @@ def test_train_1d(self, params='wmc'):
         with ignore_warnings(category=DeprecationWarning):
             g.fit(X)
             trainll = g.score(X)
-            if isinstance(g, mixture.DPGMM):
```

Comment:
DPGMM and VBGMM are both deprecated and as VBGMM inherit from DPGMM you have two warnings messages when you call VBGMM.
I've corrected that to make sure the deprecation warning is meaningfull and moreover to be sure to pass the test once I will modify the deprecation message to refer to the new BayesianGaussianMixture class.